Sources/XMTP/Client.swift

```diff
@@ -119,16 +127,13 @@ public class Client {
 		let topics: [Topic] = [.userPrivateStoreKeyBundle(account.address)]
 		let res = try await apiClient.query(topics: topics, pagination: nil)
 
+		print("Loading private keys for \(account.address)")
```

Comment:
Were these meant to be debug only? Just checking because of the envelopes one seems like it might be noisy but I don't feel strongly

Reply:
Whoops, didn't mean to leave this in.